vochain/indexer/indexer.go

```diff
@@ -374,18 +380,62 @@ func (idx *Indexer) AfterSyncBootstrap() {
 			continue
 		}
 		options := process.GetVoteOptions()
-		if err := idx.queryWithRetries(func() error {
-			return idx.db.Upsert(p, &indexertypes.Results{
-				ProcessID: p,
-				// MaxValue requires +1 since 0 is also an option
-				Votes:        indexertypes.NewEmptyVotes(int(options.MaxCount), int(options.MaxValue)+1),
-				Weight:       new(types.BigInt).SetUint64(0),
-				VoteOpts:     options,
-				EnvelopeType: process.GetEnvelopeType(),
-				Signatures:   []types.HexBytes{},
-			})
+
+		indxR := &indexertypes.Results{
+			ProcessID: p,
+			// MaxValue requires +1 since 0 is also an option
+			Votes:        indexertypes.NewEmptyVotes(int(options.MaxCount), int(options.MaxValue)+1),
+			Weight:       new(types.BigInt).SetUint64(0),
+			VoteOpts:     options,
+			EnvelopeType: process.GetEnvelopeType(),
+			Signatures:   []types.HexBytes{},
+		}
+
+		eid := process.GetEntityId()
+		currentBlockTime := time.Unix(idx.App.TimestampStartBlock(), 0)
+
+		if enableBadgerhold {
+			if err := idx.queryWithRetries(func() error {
+				return idx.db.Upsert(p, indxR)
+			}); err != nil {
+				log.Errorw(err, "cannot upsert results to db")
+				continue
+			}
+		}
+
+		compResultsHeight := uint32(0)
+
+		if _, err := queries.CreateResultProcess(ctx, indexerdb.CreateResultProcessParams{
+			ID:       indxR.ProcessID,
+			EntityID: nonNullBytes(eid),
+			// EntityIndex: int64(entity.ProcessCount),
+			StartBlock:        int64(process.GetStartBlock()),
+			EndBlock:          int64(process.GetBlockCount() + process.GetStartBlock()),
+			ResultsHeight:     int64(compResultsHeight),
+			HaveResults:       compResultsHeight > 0,
+			CensusRoot:        nonNullBytes(process.GetCensusRoot()),
+			RollingCensusRoot: nonNullBytes(process.GetRollingCensusRoot()),
+			RollingCensusSize: int64(process.GetRollingCensusSize()),
+			MaxCensusSize:     int64(process.GetMaxCensusSize()),
+			CensusUri:         process.GetCensusURI(),
+			CensusOrigin:      int64(process.GetCensusOrigin()),
+			Status:            int64(process.GetStatus()),
+			Namespace:         int64(process.GetNamespace()),
+			ModePb:            encodedPb(process.GetMode()),
+			PrivateKeys:       strings.Join(process.EncryptionPrivateKeys, ","),
+			PublicKeys:        strings.Join(process.EncryptionPublicKeys, ","),
+			CreationTime:      currentBlockTime,
+			SourceBlockHeight: int64(process.GetSourceBlockHeight()),
+			SourceNetworkID:   process.SourceNetworkId.String(),
+			Metadata:          process.GetMetadata(),
```

Comment:
I _think_ we only need to update the results columns, similar to what the old database did. the new database uses fewer tables, but you can still use SQL to only update some fields.

For that same reason, I think we want the sqlite query to be an `UPDATE ... WHERE id = ?` rather than a REPLACE; that way we don't have to provide _all_ the column values from scratch.